build/devbase/vendor.sh

```diff
@@ -44,16 +44,7 @@ function download_verify()
 }
 
 # Parallelize the builds below based on the number of cpus.
-ncpus=1
-case $(uname) in
-    Linux)
-        ncpus=$(grep processor /proc/cpuinfo | wc -l)
-	;;
-    Darwin)
-	ncpus=$(sysctl -q -n hw.logicalcpu)
-	;;
-esac
-
+ncpus=$(go run $(pwd -P)/build/devbase/ncpu.go)
```

Comment:
Copy pasta from elsewhere in this file. It actually is necessary because this script cd's a bunch of times.